eZ/Publish/Core/IO/TolerantIOService.php

```diff
@@ -105,6 +113,20 @@ public function loadBinaryFileByUri($binaryFileUri)
     }
 
     /**
+     * @param $binaryFileUri
+     *
+     * @return MissingBinaryFile
+     */
+    private function createMissingBinaryFileByUri($binaryFileUri)
```

Comment:
Well, `createMissingBinaryFile` does two things (create & load the URI). It makes a _little_ more sense to have a private. I don't mind if you move the logic from that private to the method itself.